packages/template-retail-react-app/app/pages/product-detail/index.test.js

```diff
@@ -118,7 +118,7 @@ describe('product set', () => {
                 expect(within(modal).getByText(/items added to cart/i)).toBeInTheDocument()
             },
             // Seems like rendering the modal takes a bit more time
-            {timeout: 5000}
+            {timeout: 10000}
```

Comment:
There was some feedback about investigating the root cause.. perhaps the modal is the culprit here.

So I did look into that and could not find any fault with the modal rendering. Switching to the modal's own tests and using `console.time` there, I saw the time to be quite fast already.